umm-spec-lib/src/cmr/umm_spec/metadata_specification.clj

```diff
@@ -1,17 +1,24 @@
 (ns cmr.umm-spec.metadata-specification
   "Functions related to the MetadataSpecification node common to many of the umm
    models. Eventually all models will include this node, so a common set of
-   functions is needed."
+   functions is needed.
+   "
   (:require
    [cmr.umm-spec.versioning :as ver]))
 
 (def types
   "This is a mapping between collection symbol types to names found in metadata"
-  {:collection "UMM-C"  ; currently does not have this node
+  {:collection "UMM-C"  ; Currently does not have this node
```

Comment:
Is that comment still applicable?

Reply:
it is still correct, I don't know where it is on the schedule, but we are running out of small schemas to change, so it's day is coming

Reply:
I'm adding the word "Reserved" to the comment.